pageserver/src/task_mgr.rs

```diff
@@ -135,6 +135,19 @@ pub static BACKGROUND_RUNTIME: Lazy<Runtime> = Lazy::new(|| {
         .expect("Failed to create background op runtime")
 });
 
+pub(crate) static RUNTIME_WORKER_THREADS: Lazy<usize> = Lazy::new(|| {
+    // force init and thus panics
+    drop(BACKGROUND_RUNTIME.handle());
+    // replicates tokio-1.28.1::loom::sys::num_cpus which is not available publicly
+    // tokio would had already panicked for parsing errors or NotUnicode
+    //
+    // this will be wrong if any of the runtimes gets their worker threads configured to something
+    // else, but that has not been needed in a long time.
+    std::env::var("TOKIO_WORKER_THREADS")
```

Comment:
it seems that this part of code does not depend on `BACKGROUND_RUNTIME`, and therefore it would be safe to not initializing `BACKGROUND_RUNTIME` in this `Lazy` constructor?

Reply:
This is an env variable tokio looks at when deciding on how many worker threads.

We depend on tokio panicking if the env variable is not unicode or cannot be parsed.

Reply:
Just realized that we don't necessary start all runtimes on `main()` but luckily for this one we do. Should probably add eager init which would set the order of threads as well (as in for gdb `info threads`).